scripts/reborn_webui_v2_live_qa/slack_helpers.py

```
@@ -598,21 +871,337 @@ def _slack_auth_test(config_text: str, extra_env: dict[str, str]) -> dict[str, o
         result["needed"] = payload.get("needed")
     return result
 
 
+def _slack_env_field(
+    names: list[str],
+    extra_env: dict[str, str] | None = None,
+) -> tuple[str, str] | None:
+    return _first_env_value(names, extra_env)
+
+
+def _slack_personal_auth_preflight(
```

Ruff (0.15.20)

[warning] Too many branches (17 > 12)

(PLR0912)